keyboards/bminiex/readme.md

```diff
@@ -0,0 +1,14 @@
+B.mini
+========
+
+A 75% keyboard with RGB
```

Comment:
Should specify something along the lines of this being a 75% + Numpad layout.

Reply:
I described it as "a compact fullsize keyboard with RGB" since it really has all keys to make a fullsize but the layout is different (clusters are squashed together).

Reply:
That'll work.